docs/operator/overview/README.md

```diff
@@ -0,0 +1,318 @@
+# Polymesh Operator - Overview
+
+### **Strictly Confidential**
```

Comment:
just another of https://github.com/PolymathNetwork/polymesh-tools/pull/4/files#r554869279